### DISCUSSION TITLE
Feature Request: Please add Wild Card function to Search Window, and Sort the content.

### DISCUSSION THREAD

**Discussion by wa0uwh** [thomasnordquist/MQTT-Explorer#846] [Ideas]
It would be very useful if Wild Card character could be used in the Search window, similar to that used by Mqtt Subscribers Apps.   

Also, it would be very useful if the items within a low level list could be Sorted, with very long lists, it is sometimes very difficult to organize (trouble shoot) the details for several devices/sensors, etc.

Thanks for your efforts, I use "Mqtt Explorer" many times each day, again Thanks.
-Eldonb

**Comment by matanmaz** [comment]
what do you mean by wild card? do you refer to the MQTT wildcard or just to be able to use regex in the search?
